src/java_bytecode/jar_file.h

```diff
@@ -15,27 +15,27 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <string>
 #include <vector>
 #include <map>
+#include <regex>
+#include <util/message.h>
 
-class jar_filet
+class jar_filet:public messaget
 {
 public:
   jar_filet():mz_ok(false) { }
-
-  inline explicit jar_filet(const std::string &file_name)
-  {
-    open(file_name);
-  }
+  inline explicit jar_filet(const std::string &file_name) { }
```

Comment:
Though: what does this constructor even do!? (Except for failing to initialize mz_ok.)